R/nse-inject.R

```diff
@@ -400,55 +440,62 @@ NULL
 #' as_label(letters)
 #'
 #' @export
-englue <- function(x) {
-  check_string(x)
+englue <- function(x,
+                   env = caller_env(),
+                   error_call = current_env(),
```

Comment:
I don't have strong opinions, but do you have a reason for using `current_env()` over `caller_env()`?

`englue()` will pretty much always be wrapped inside another function right? Should it default to reporting that function as the location of any issues?

Reply:
Most of the time `englue()` will be used as part of tidy eval to program against dplyr or tidyr. In these cases, I think it makes sense to match input errors to the functions that the user used, which is `englue()`.